### PR TITLE
fix: "required" markers weren't colored correctly

### PR DESCRIPTION
**Pull request by marksdk**

### CODE REVIEW

File: collections/_api/v1/index.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ All Geodata BaseTypes have some common keys that is available for all, and then
 
 #### BaseTypeProperties for Venue
 
-* `Administrativeid [string] - <span class="red">required</span>`
+* `Administrativeid [string]` - <span style="color:red">*required</span>
```
Comment (PelleRavn) [inline collections/_api/v1/index.md]: There is a `class` called `red`, why not still use that, and just only move the quote `'` not to display the HTML?

Comment (marksdk) [reply collections/_api/v1/index.md]: @PelleRavn Because there is no class called `red` 🙂

Comment (PelleRavn) [reply collections/_api/v1/index.md]: There is one in tachyons.min.css, that is included.

image

**Comment by marksdk** [comment]
@PelleRavn Try again 🤡

**Pull request merged**